### DISCUSSION TITLE
Turn off "Show Labels on Hover" seems not working

### DISCUSSION THREAD

**Discussion by tyeen** [typefully/minimal-twitter#98] [Q&A] [answered]
I'm not sure if this is the behavior by design. With "Show Labels" on and "Show Labels on Hover" off, the navigation label does not show on window size < 1450px.

**Comment by ghost** [comment, marked as answer]
The desktop breakpoint is `1265px` so if you're below that then there won't be labels

**Reply by tyeen** [reply]
I'm not sure if I understand correctly, but here is the screenshot not showing labels with body size 1449px.
<img width="1561" alt="Screen Shot 2022-03-21 at 23 58 57" src="https://user-images.githubusercontent.com/788863/159289084-4956f295-67c0-4ffb-b5b9-92407bf4ea2e.png">

Then another one showing labels with body size 1450px.
<img width="1562" alt="Screen Shot 2022-03-21 at 23 59 12" src="https://user-images.githubusercontent.com/788863/159289146-305531b6-cdb0-45c5-a1b5-824f4e8b56d4.png">

**Reply by ghost** [reply]
Thanks for the further detail! Fixed in `4.1.10`: https://github.com/thomaswang/minimal-twitter/releases/tag/v4.1.10. This breakpoint was added to prevent overlap on wider main column pages like `twitter.com/explore` — but you can turn off labels in that case if it bothers you.

**Reply by tyeen** [reply]
Yes, 4.1.10 confirmed!🎉
